### ISSUE TITLE
label color change

### ISSUE THREAD

**Issue by slaygun** [Moon-0xff/gnome-mpris-label#93]
hey can u lmk how i could change the color of the label text which gets displayed

**Comment by Batwam** [comment]
hi,

we don't currently have an option for this but you can update `extensions.js` manually around row 485 as follows (example here with red font):
```js
	_setText() {
		try{
			if(this.player == null || undefined)
				this.label.set_text("")
			else {
				this.label.set_text(buildLabel(this.players,this.settings));
				this.label.set_style('color: #FF0000');
			}
		}
		catch(err){
			log("Mpris Label: " + err);
			this.label.set_text("");
		}
	}
```
![image](https://github.com/Moon-0xff/gnome-mpris-label/assets/56710655/ea2ebfce-0f66-4590-879b-3850f072e96b)

**Comment by slaygun** [comment]
omg thank u so much im not well versed with javascript so i was having a pretty hard time

**Comment by slaygun** [comment]
![image_2024-01-31_152846243](https://github.com/Moon-0xff/gnome-mpris-label/assets/73815160/23786220-fb87-4c4b-ae9f-57159ace3875)
it works!

**Comment by Batwam** [comment]
No problem, it's a good suggestion and I was actually thinking this weekend about allowing to change things like the font. 

I'll see if I can integrate an option in the Preferences to allow the user to pick a colour easily.

**Comment by Batwam** [comment]
@slaygun could you please test the solution in #94 ?

**Comment by slaygun** [comment]
@Batwam what is the command to get that branch

**Comment by Batwam** [comment]
Sorry, git clone the repo, then get the branch by doing `git checkout font_colour`

**Comment by slaygun** [comment]
![image](https://github.com/Moon-0xff/gnome-mpris-label/assets/73815160/4f386555-df1f-4f48-b339-4f796b95afc6)
yes its working!

**Comment by Batwam** [comment]
ok, that's great. This should allow you to try different colours without having to manually change the code and without having to restart the extension all the time. Let me see if I can integrate a button to select a colour more easily...
![color-button](https://github.com/Moon-0xff/gnome-mpris-label/assets/56710655/a42fa3d2-6af6-4f7a-bf33-223cda25d2b8)

**Comment by Batwam** [comment]
@slaygun I have updated the Preferences to include a button to select the colour. Would you be able to download the latest code and try it?
![image](https://github.com/Moon-0xff/gnome-mpris-label/assets/56710655/6b91d266-5a15-48e5-8abf-774eb9d6d961)

**Comment by slaygun** [comment]
i did and its working :+1:

**Comment by Batwam** [comment]
Ok, that's great to hear. 

i'll give it a couple of days to let things settle and merge if all goes well. Let me know if you face any issues. Are you using gnome 45?

**Comment by slaygun** [comment]
Okaaay will do. Yes, I'm using gnome 45

**Comment by Batwam** [comment]
Closing this as the requested feature has now been implemented in #94
